### PR TITLE
Update the docs to include CC global setting

### PR DESCRIPTION
**Pull request by vagababov**
CC can be set globally via config-defaults CM.

/assign @markusthoemmes @abrennan89

### CODE REVIEW

File: docs/serving/configuring-autoscaling.md [hunk 1]
```diff
@@ -221,15 +221,26 @@ spec:
...
 
-* **Global key:** n/a
+* **Global key:** `container-concurrency` (`config-defaults` config map)
 * **Per-revision spec key:** `containerConcurrency`
 * **Possible values:** integer
 * **Default:** `0`, which means unlimited
 
 **Example:**
 {{< tabs name="container-concurrency" default="Per Revision" >}}
+{{% tab name="Global (ConfigMap)" %}}
```
Comment (markusthoemmes) [inline docs/serving/configuring-autoscaling.md:233]: Please put this below the per-revision config for uniformity.

File: docs/serving/configuring-autoscaling.md [hunk 2]
```diff
@@ -221,15 +221,26 @@ spec:
 
 #### Hard Limit (containerConcurrency)
 
-The hard limit has no global setting and is only specifiable per-revision. Its per-revision setting is also not an annotation but is actually present on the revision's spec itself as `containerConcurrency`. Its default value is "0", which means an unlimited number of requests are allowed to flow into the replica. A value above "0" specifies the exact amount of requests allowed to the replica at a time.
+The hard limit has its global setting in the `config-defaults` config map and can also be specifiable per-revision. Its per-revision setting is also not an annotation but is actually present on the revision's spec itself as `containerConcurrency`. Its default value is "0", which means an unlimited number of requests are allowed to flow into the replica. A value above "0" specifies the exact amount of requests allowed to the replica at a time.
```
Comment (markusthoemmes) [inline docs/serving/configuring-autoscaling.md:224]:
```suggestion
The hard limit has its global setting in the `config-defaults` config map and can also be specified per-revision. Its per-revision setting is not an annotation but is actually present on the revision's spec itself as `containerConcurrency`. Its default value is "0", which means an unlimited number of requests are allowed to flow into the replica. A value above "0" specifies the exact amount of requests allowed to the replica at a time.
```

Comment (vagababov) [reply docs/serving/configuring-autoscaling.md]: Hm :) I kept this word, I thought you wanted it for a reason 🐌

File: docs/serving/configuring-autoscaling.md [hunk 3]
```diff
@@ -245,6 +245,17 @@ spec:
         - image: gcr.io/knative-samples/helloworld-go
 ```
 {{< /tab >}}
+{{% tab name="Global (ConfigMap)" %}}
```
Comment (markusthoemmes) [inline docs/serving/configuring-autoscaling.md:248]: Can you add a "Global (Operator)" too? :)

Comment (vagababov) [reply docs/serving/configuring-autoscaling.md:248]: OKay :)

**Comment by abrennan89** [comment]
lgtm!

**Pull request merged**